gapic-generator-cloud/templates/cloud/service/client/_credentials.erb

```diff
@@ -3,7 +3,7 @@
 require "googleauth"
 <% end %>
 class <%= service.credentials_name %> < Google::Auth::Credentials
-  SCOPE = ["https://www.googleapis.com/auth/cloud-platform"]
+  SCOPE = <%= service.client_scopes.inspect %>
```

Comment:
Can we freeze the assigned value of `SCOPE` here (and similarly in `gapic-generator`)?

Reply:
The Rubocop formatting will add it. This will go away eventually, so I don't really care to add a `.freeze` here.

Reply:
Sounds good. Want to add a comment explaining where this is headed/ why it will go away?